pkg/agent/proxy/proxier.go

```diff
@@ -847,6 +847,9 @@ func (p *proxier) Run(stopCh <-chan struct{}) {
 		go p.serviceConfig.Run(stopCh)
 		if p.endpointSliceEnabled {
 			go p.endpointSliceConfig.Run(stopCh)
+			if p.topologyAwareHintsEnabled {
+				go p.nodeConfig.Run(stopCh)
```

Comment:
I am wondering is there any e2e for this? or is possible to avoid such issue when dependent controller is not started?
cc @tnqn

Reply:
I guess sig-network e2e has tests related to topologyAwareHints